lib/rabbitmq/cli/ctl/commands/list_consumers_command.ex

```diff
@@ -27,12 +27,12 @@ defmodule RabbitMQ.CLI.Ctl.Commands.ListConsumersCommand do
   def aliases(), do: [t: :timeout]
 
   @info_keys ~w(queue_name channel_pid consumer_tag
-                ack_required prefetch_count single_active arguments)a
+                ack_required prefetch_count active activity_status arguments)a
 
   def info_keys(), do: @info_keys
 
   def merge_defaults([], opts) do
-    {Enum.map(@info_keys -- [:single_active], &Atom.to_string/1), Map.merge(%{vhost: "/", table_headers: true}, opts)}
+    {Enum.map(@info_keys -- [:active, :activity_status], &Atom.to_string/1), Map.merge(%{vhost: "/", table_headers: true}, opts)}
```

Comment:
Should activity be in the default arguments?

Reply:
Not sure. Those 2 fields matter when single active consumer is on or when investigating an issue with quorum queues. To me these are specific cases, not the most common ones, that's why the 2 fields are not in the default ones. I was also thinking that adding them (or one) would bring up questions for cases where they don't matter. What do you think @kjnilsson?

Reply:
If most users won't need them I'd say let's leave them out.

Reply:
Yes, the `activity_status` field is only needed in case of investigations, but the `active` field may be useful in general to detect if there is a need for investigation.

Reply:
This is reasonable.

Reply:
OK, the active field is now in the defaults.